README.md

```diff
@@ -76,15 +87,35 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
+app.get('/posts', (req, res) => {
+  console.log('index or show')
+})
+
+app.post('/posts', (req, res) => {
+  console.log('create')
+})
 
+app.put('/posts/:title', (req, res) => {
+  console.log('update')
+})
+
+app.delete('/posts/:title', (req, res) => {
+  console.log('destroy')
+})
 ```
```

Comment:
:+1: although in order to do this we would need to use something like the method override middleware